packages/shell-api/src/shell-api.spec.ts

```diff
@@ -157,7 +157,7 @@ describe('ShellApi', () => {
       it('fails with no arg', async() => {
         serviceProvider.platform = ReplPlatform.CLI;
         try {
-          // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
is there any other way around `@ts-ignore` here? would be nice to extend the ban to `@ts-ignore` to tests too.

Reply:
yeah, this can also use `as any` … kind of comes down to the same thing, I think :)

Reply:
`@ts-ignore` disables everything, is not going to make much of a difference since have `eslint` catching  issues too, but with `@ts-ignore` for example you could use variables that are not defined anywhere, like `bla + 123` would be compiled just fine.

Reply:
but yeah is more of a 'let's shut down the door to bad habits' than something having any practical effect.

Reply:
ok, done :)